core/src/main/java/org/opensearch/sql/expression/function/FunctionPropertiesTestConfig.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.function;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FunctionPropertiesTestConfig {
```

Comment:
Move TestConfig to test?

Reply:
Moved in ea4588e3ca86157b709bd7184a27edc79253e8f5